Resources/Audio/_CD/Voice/Talk/attributions.yml

```
@@ -0,0 +1,4 @@
+- files: ["moth.ogg, moth_ask.ogg, moth_exclaim.ogg"]
```

The attribution is incorrect, it's saying it's for one file named "moth.ogg, moth_ask.ogg, moth_exclaim.ogg"
```suggestion
- files: ["moth.ogg", "moth_ask.ogg", "moth_exclaim.ogg"]
```